### ISSUE TITLE
tfkanSome doubts about tfkan

### ISSUE THREAD

**Issue by Summer-77** [ZPZhou-lab/tfkan#10]
Hello, I would like to ask you about tfkan in which I input my training data and it goes to fit it, but when I go to test it after the training is over, does the fitted function still exist? As we all know that tfkan is a network similar to MLP, do we need to train it before using it and then embedding it in our own model. I hope you can answer this doubt for me.

**Comment by ZPZhou-lab** [comment]
Yes, the fitted function still exist after you train the model. All of the `tfkan` modules have the same behavior as modules in `Tensorflow`. Therefore, you can build your model using `tfkan` and other tensorflow modules, train it and use it at testing time.

**Comment by Summer-77** [comment]
I've heard that kan is a bit friendlier when it comes to timing signals, but it doesn't seem to work very well when I add it, is it because I have a problem with the parameter settings?
